voronoi_game.py

```diff
@@ -103,29 +107,15 @@ def __get_player_move(self):
     data = client_response.split()
     return int(data[0]), int(data[1])
 
+  def compute_pull(self, row, col):
+    return np.reciprocal(np.square(self.row_numbers - row) + np.square(self.col_numbers - col) + 0.1e-30)
```

Comment:
do you think here you could do something like:
```python
temp = np.square(self.row_numbers - row) + np.square(self.col_numbers - col)
temp[row][col] += 1 #since this won't be used for computing scores anyway as there's a stone on it
return np.reciprocal(temp)
```

??